docusaurus/docs/learn/quickstarts/network/local-no-docker.md

```diff
@@ -31,7 +31,7 @@ Running the latest version of Ziti locally is as simple as running this one comm
 
 This script will perform an 'express' install of Ziti which does the following:
 
-* download the latest version of the Ziti components (`ziti`, `ziti-controller`, `ziti-edge-router`, `ziti-tunnel`)
+* download the latest version of the Ziti components (`ziti`, `ziti-controller`, `ziti-router`, `ziti-tunnel`)

```

Comment:
now-a-days we don't need `ziti-controller` and `ziti-router` nor `ziti-tunnel`. can you find/replace those with just "download the latest `ziti` binary"?

Reply:
Will fix